src/Fetch.ml

```diff
@@ -379,6 +379,29 @@ module Response = struct
   external clone : t = "clone" [@@bs.send.pipe: t]
 end
 
+(* This belongs to XHR spec and will probably be moved to another repo in the future *)
+
+module Blob = struct
```

Comment:
wonder if we need this `Blob` module given that there's a `type blob` above in this file.